### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by ericmarcos**
## Description

Update README

## Context

Information was a bit outdated and some links/images broken

### CODE REVIEW

**Review by vanbasten17 — APPROVED** [review]
Looking pretty cool!  😎 Should we add it also to @botonic/react's README, which was quite similar?

File: README.md [hunk 1]
```diff
@@ -71,26 +90,23 @@ $> botonic serve
 $> botonic deploy
 ```
 
-Check out the [Getting Started Tutorial](https://botonic.io/docs/getting-started.html) is a step-by-step guide to start building high quality conversational apps.
-
-[See the docs for more information](https://botonic.io/docs.html).
+Check out the [Getting Started Tutorial](https://botonic.io/docs/getting-started) is a step-by-step guide to start building high quality conversational apps.
```
Comment (annehubtype) [inline README.md:93]: Check out the [Getting Started Tutorial](https://botonic.io/docs/getting-started): a step-by-step guide to start building high quality conversational apps.

File: README.md [hunk 2]
```diff
@@ -12,36 +13,54 @@
...
+With Botonic you can focus on creating the best conversational experience for your users instead of dealing with different messaging APIs, AI/NLP complexity or managing and scaling infrastructure.
+It also comes with a battery of plugins so you can easily integrate popular services into your project, for instance:
+
+- Analytics ([Google Analytics](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-google-analytics), [Segment](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-segment), [Dashbot](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-dashbot))
+- CMS ([Contentful](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-contentful), Sanity)
+- NLU ([Dialogflow](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-dialogflow), [Watson](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-watson), [Luis](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-luis), [Inbenta](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-inbenta))
+
+_If you'd like to see more plugins/integrations please submit an issue or a pull request_
```
Comment (annehubtype) [inline README.md:54]: _If you'd like to see more plugins/integrations, please submit an issue or a pull request_

**Review by annehubtype — APPROVED** [review]
Very nice!

**Pull request merged**